Code/src/pages/Tab1.tsx

```diff
@@ -1,107 +1,65 @@
-import {
-  IonContent,
-  IonHeader,
-  IonPage,
-  IonTitle,
-  IonToolbar,
-  IonCard,
-  IonCardContent,
-  IonCardHeader,
-  IonCardSubtitle,
-  IonCardTitle,
-  IonThumbnail,
-  IonSearchbar,
-  IonItem,
-} from "@ionic/react";
-import React from "react";
-import "./Tab1.css";
+import React, { createRef } from 'react';
+import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar, IonSearchbar, IonList, IonItem, IonLabel, IonCard, IonCardHeader, IonCardTitle, IonCardSubtitle, IonCardContent, IonButton, IonInfiniteScroll, ScrollDetail,IonInfiniteScrollContent} from '@ionic/react';
```

Comment:
Do you have the "prettier" addon? It helps format files so they don't get too wide